api/secpolnodestatus/v1alpha1/secpolnodestatus_types.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ProfileState defines the state that the profile is in. A profile in this context
+// refers to a SeccompProfile or a SELinux profile, the states are shared between them
+// as well as the management API.
+type ProfileState string
+
+const (
+	// The profile is pending installation.
+	ProfileStatePending ProfileState = "Pending"
+	// The profile is being installed.
+	ProfileStateInProgress ProfileState = "InProgress"
+	// The profile was installed successfully.
+	ProfileStateInstalled ProfileState = "Installed"
+	// The profile is being removed and is currently terminating.
+	ProfileStateTerminating ProfileState = "Terminating"
+	// The profile couldn't be installed.
+	ProfileStateError ProfileState = "Error"
+	// When adding new statuses, remember to also adjust the LowerOfTwoStates function.
+)
+
+// Common labels of the node status objects.
+const (
+	// Identifies the policy by name so that the admin can list all node statuses for a certain policy.
+	StatusToPolLabel = "spo.x-k8s.io/policy-name"
+	// Identifies the node on which the policy is installed so that the admin can list policies per node.
+	StatusToNodeLabel = "spo.x-k8s.io/node-name"
+	// Allows the admin to filter out node statuses with a certain state (e.g. show me all that failed).
+	StatusStateLabel = "spo.x-k8s.io/policy-state"
+	// The kind of policy so that the admin can filter only e.g. all selinux policy statuses.
+	StatusKindLabel = "spo.x-k8s.io/policy-kind"
+)
```

Comment:
Is there a reason to use "policy" here over "profile"?

Reply:
not really, fixed